js/components/graph/__tests__/FocusBar.test.js

```diff
@@ -0,0 +1,24 @@
+import { fireEvent} from "@testing-library/react";
+import TestFocusBar from "./TestFocusBar";
+import TestContainer from "./TestContainer";
+
+describe("FocusBar", () => {
+  it("should match shallow snapshot", async () => {
+    const focusBar = await TestFocusBar.build();
+    expect(focusBar).toMatchSnapshot();
+  });
+
+  it("Focusbar should be disabled for non-computer science graphs", async () => {
```

Comment:
Be consistent with capitalization of "FocusBar". (Review this throughout your test descriptions.)